packages/nest/src/schematics/application/application.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  apply,
+  chain,
+  externalSchematic,
+  mergeWith,
+  move,
+  Rule,
+  SchematicContext,
+  template,
+  Tree,
+  url
+} from '@angular-devkit/schematics';
+import { join, normalize, Path } from '@angular-devkit/core';
+import { Schema } from './schema';
+import { toFileName } from '@nrwl/schematics/src/utils/name-utils';
+
+interface NormalizedSchema extends Schema {
+  appProjectRoot: Path;
+}
+
+function addMainFile(options: NormalizedSchema): Rule {
+  return (host: Tree) => {
+    host.overwrite(
```

Comment:
It might be easier to use a `files` directory + `MergeStrategy.Overwrite` so that we can do better templating. :woman_shrugging: